tests/ClientTest.php

```diff
@@ -2,44 +2,61 @@
 
 namespace Solarium\Tests;
 
+use Composer\InstalledVersions;
 use PHPUnit\Framework\TestCase;
 use Solarium\Client;
 
 class ClientTest extends TestCase
 {
+    /**
+     * The version tag we use for testing within github actions.
+     *
+     * @var string
+     */
+    protected static $versionTag = '76.5.4';
```

Comment:
We currently have test failures for release tags, namely 6.2.6.
I think we should use ```InstalledVersions::getPrettyVersion('solarium/solarium')``` and use its value if it is a valid semantic version number.